src/win_mpiconstants.jl

```diff
@@ -0,0 +1,78 @@
+const MPI_BYTE = Int32(0x4c00010d)
```

Comment:
If this file is auto-generated, then please add a pointer how this was done.

If it was manually generated, then (a) thanks for the hard work, and (b) please point to a description (or add a comment describing) where the constants come from. Presumably, these can change, e.g. in a newer version of Microsoft's MPI.